plugins/kubernetes/app/decorators/project_decorator.rb

```diff
@@ -54,6 +54,10 @@ def name_for_label
     name.parameterize('-')
   end
 
+  def deploy_watcher_actor_name
```

Comment:
Wouldn't it be cleaner if the deploy watcher handled generating it's own name (like the cluster watchers)? This way the project code needs to be kinda aware of the existence of watchers.